README.md

```diff
@@ -239,6 +239,31 @@ export default class World extends ActorWorld {
 setWorldConstructor(World)
 ```
 
+If you are using ES Modules (by setting `"type": "module"` in `package.json`), you need to
+change ypur `World` constructor, and add a `Begin` step:
+
+```typescript
+import { setWorldConstructor } from '@cucumber/cucumber'
+import { ActorWorld, defineActorParameterType, Action } from '@cucumber/screenplay'
+import { InboxMessages, Shout, StartSession } from './tasks/types'
+export default class World extends ActorWorld {
+  // These tasks will be loaded automatically
+  public startSession: StartSession
+  public shout: Shout
+  public inboxMessages: InboxMessages
+  constructor(props: IWorldOptions) {
+    super({ ...props, packageType: 'module' })
+  }
+}
+setWorldConstructor(World)
+
+Begin(() => {
```

Comment:
Would it be possible to write this hook from within the `ActorWorld` to save the user having to have this extra configuration step?

Reply:
I couldn't find a way to do that, but I'd be happy to go through it in a pairing session!